examples/fc_config.go

```diff
@@ -0,0 +1,91 @@
+package main
+
+import (
+	"fmt"
+	"github.com/HewlettPackard/oneview-golang/ov"
+	"github.com/HewlettPackard/oneview-golang/utils"
+)
+
+func main() {
+	var (
+		ClientOV    *ov.OVClient
```

Comment:
If the configuration file is used, this example will throw a compilation error saying unused ClientOV. Can you put the declaration part of this variable in the comment section mentioned below, so that if the user doesn't want to the config file, user can uncomment and use and vice versa, user can use config file without any error.